### ISSUE TITLE
words cloud text overlap 詞雲文字重疊

### ISSUE THREAD

**Issue by red010182** [antvis/g2#256]
* **G2 Version**: 3.0.0
* **Platform**: OSX Chrome/Safari

* **CodePen Link**: https://codepen.io/alstontsao/pen/NwVBOV

兩個問題：
1. 旋轉角度設為0時，詞會重疊

```
      rotate: 0, //cfg.origin._origin.rotate,
```

![](https://i.imgur.com/p6P9I1L.png)

2. 
```
dv.transform({
    ...
    fontSize(d) {  
         // 此函數毫無作用，就算return 0或return 10000都不影響字體大小
    }
})

```

**Comment by leungwensen** [comment]
Thank you for submitting this issue, we are working on it. This will be included in the release of `@antv/data-set@0.8.0` next week.

**Comment by leungwensen** [comment]
Done

@antvis/g2@3.0.2 &&  @antv/data-set@0.8.0

[word-cloud](https://antv.alipay.com/zh-cn/g2/3.x/demo/other/word-cloud.html) [word-cloud with image mask](https://antv.alipay.com/zh-cn/g2/3.x/demo/other/word-cloud-mask.html)
